src/Models/Location.php

```diff
@@ -33,10 +33,11 @@ class Location extends BaseModel
         'php_tz',
     ];
 
-    public function getRouteKeyName()
+    /*@todo since we remove slug, what would be the field  binding the model?*/
+    /*public function getRouteKeyName()
```

Comment:
@drewroberts since this will be handle by the blog package and there's no `slug` field, I dont know if delete the function